src/views/GovernanceReviewTeam/Dates/index.tsx

```diff
@@ -13,31 +16,33 @@ import FieldGroup from 'components/shared/FieldGroup';
 import Label from 'components/shared/Label';
 import TextField from 'components/shared/TextField';
 import { AnythingWrongSurvey } from 'components/Survey';
-import { AppState } from 'reducers/rootReducer';
-import { saveSystemIntake } from 'types/routines';
-import { SubmitDatesForm, SystemIntakeForm } from 'types/systemIntake';
+import { SubmitDatesForm } from 'types/systemIntake';
 import flattenErrors from 'utils/flattenErrors';
 import { DateValidationSchema } from 'validations/systemIntakeSchema';
 
-const Dates = ({ systemIntake }: { systemIntake: SystemIntakeForm }) => {
+const Dates = ({ systemIntake }: { systemIntake: SystemIntake }) => {
   const { systemId } = useParams<{ systemId: string }>();
-  const dispatch = useDispatch();
   const history = useHistory();
   const { t } = useTranslation();
-  const intakeIsLoading = useSelector(
-    (state: AppState) => state.systemIntake.isLoading
+  const [mutate, mutationResult] = useMutation<UpdateSystemIntakeReviewDates>(
```

Comment:
I don't quite know what it buys us (still trying to discern that), but you might add a second type arg here

`useMutation<UpdateSystemIntakeReviewDates, UpdateSystemIntakeReviewDatesVariables>`